src/components/TransactionConfirmationModal/index.tsx

```diff
@@ -5,7 +5,7 @@ import { Text } from 'rebass'
 import styled, { useTheme } from 'styled-components'
 import Circle from '../../assets/images/blue-loader.svg'
 import { useActiveWeb3React } from '../../hooks/useActiveWeb3React'
-import { ExternalLink } from '../../theme'
+import { ExternalLink, theme as TwTheme } from '../../theme'
```

Comment:
define

```typescript
export type TwTheme =  ReturnType<typeof theme>
```

 at `theme/index.tsx` and use it everywhere

Reply:
done